src/addresses.rs

```diff
@@ -4,9 +4,76 @@ use mimirsbrunn::admin_geofinder::AdminGeoFinder;
 use mimirsbrunn::labels::format_addr_name_and_label;
 use mimirsbrunn::labels::format_street_label;
 use mimirsbrunn::utils::find_country_codes;
+use reqwest::StatusCode;
+use serde::Deserialize;
 use std::ops::Deref;
 use std::sync::Arc;
 
+/// Check if a mimir address originates from OSM data.
+pub fn is_osm_addr(addr: &mimir::Address) -> bool {
+    match addr {
+        mimir::Address::Addr(addr) => addr.id.contains(":osm:"),
+        mimir::Address::Street(st) => st.id.contains(":osm:"),
+    }
+}
```

Comment:
Argh, we lost a check somewhere. This function is no longer used.

Reply:
Grmf, how did I miss this ... It is supposed to be there: https://github.com/QwantResearch/fafnir/pull/47/files#diff-9b894eb37fb1dd6f240a692c6167c7dfR223